src/frontend/src/utils/iiConnection.ts

```diff
@@ -101,6 +102,8 @@ export type { ChallengeResult } from "../../generated/internet_identity_types";
 
 export interface IdentifiableIdentity extends SignIdentity {
   rawId: ArrayBuffer;
+
+  getAuthenticatorAttachment(): AuthenticatorAttachment | undefined;
```

Comment:
Do you think `IdentifiableIdentity` is still a good name if the interface also takes care of providing authenticator attachment data?

Reply:
Ideally, it would be called `WebAuthnIdentity` but then it clashes with the `agent-js` type. Maybe `AuthenticatorIdentity`? Or `WebAuthnSignIdentity`?

Reply:
I like `AuthenticatorIdentity`!

Reply:
Note that you can also `import { WebAuthnIdentity as OtherName } from ...` although it can be a little confusing

Reply:
> Note that you can also import { WebAuthnIdentity as OtherName } from ... although it can be a little confusing

Yes, but I didn't do that to avoid the confusion. I now called it `IIWebAuthnIdentity` and added an explanation on how it relates to  `WebAuthnIdentity`.